README.md

```diff
@@ -34,6 +34,10 @@ Polaris Catalog is open source under an Apache 2.0 license.
 - ⭐ Star this repo if you’d like to bookmark and come back to it! 
 - 📖 Read the <a href="https://www.snowflake.com/blog/polaris-catalog-open-source/" target="_blank">announcement blog post<a/> for more details!
 
+## Development
+
+See [CONTRIBUTING](CONTRIBUTING.md) for contribution requirements.
```

Comment:
Is this link going to work in the README page?

Reply:
Yes it works. You can verify it here https://github.com/krvikash/polaris/tree/krvikash/add-development-section-readme?tab=readme-ov-file#development